src/Providers/MiddlewareProvider.php

```diff
@@ -22,9 +23,12 @@ class MiddlewareProvider extends AbstractServiceProvider
      */
     public function register()
     {
-        $this->container->share(MultiTenantFamilyHydrator::class, function(){
-            $entityManager = $this->container->get(EntityManager::class);
+        $this->container->share(MultiTenantFamilyHydrator::class, function () {
+            $entityManager = $this->container->get(EntityManagerInterface::class);
+
             return new MultiTenantFamilyHydrator($entityManager->getRepository(Family::class));
         });
+
+        $this->container->share(FamilyHydrator::class);
```

Comment:
Since the `FamilyHydrator` is being instantiated in the RouterProvider, it probs doesn't need to be provided here?

Reply:
:+1: